### PR TITLE
A few patches I did to make my program work with these controls the way I wanted.

### PR DESCRIPTION
**Pull request by AraHaan**
And yes expect a change pull requesting disabled state on any control I am interested in here as well.

### CODE REVIEW

**Comment by sylveon** [comment]
Why are you pull requesting ShareX for changes you want for *your* program but useless for us, and giving on us the burden of maintaining this code?

File: ShareX.HelpersLib/Controls/BlackStyle/BlackStyleButton.cs [hunk 1]
```diff
@@ -33,120 +33,263 @@
...
+        private LinearGradientBrush backgroundHoverBrush;
+        private LinearGradientBrush backgroundDisabledBrush;
+        private LinearGradientBrush innerBorderBrush;
+        private LinearGradientBrush innerBorderDisabledBrush;
+        private Pen innerBorderPen;
+        private Pen innerBorderDisabledPen;
+        private Pen borderPen;
+        private Pen borderDisabledPen;
+
+        public ThemedButton()
```
Comment (AraHaan) [inline ShareX.HelpersLib/Controls/BlackStyle/BlackStyleButton.cs]:
```suggestion
        public BlackStyleButton()
```

File: ShareX.HelpersLib/Controls/BlackStyle/BlackStyleLabel.cs [hunk 2]
```diff
@@ -34,123 +34,118 @@ namespace ShareX.HelpersLib
     public class BlackStyleLabel : Control
     {
         private string text;
+        private ContentAlignment textAlign;
+        private Color textShadowColor;
+        private bool drawBorder;
+        private bool autoEllipsis;
+        private bool wordWrap = true;
+
+        public ThemedLabel()
```
Comment (AraHaan) [inline ShareX.HelpersLib/Controls/BlackStyle/BlackStyleLabel.cs]:
```suggestion
        public BlackStyleLabel()
```

**Comment by AraHaan** [comment]
I wanted for you guys to also have the fixes in case you guys wanted to use the changes as well.

**Comment by AraHaan** [comment]
hmm

**Comment by AraHaan** [comment]
hmm

**Comment by AraHaan** [comment]
hmm

**Pull request closed without merging**